### PR TITLE
pings: make it work with Cloud Run and `sg msp`

### PR DESCRIPTION
**Pull request by unknwon**
Small things identified while trying using `sg msp` to deploy the service through Cloud Run.

## Test plan

Manually tested to have a service up and running in a test project.

### CODE REVIEW

File: dev/managedservicesplatform/managedservicesplatform.go [hunk 1]
```diff
@@ -79,12 +79,13 @@ func (r *Renderer) RenderEnvironment(
 			"environment": env.ID,
 			"msp":         "true",
 		},
+		ProjectFolderID: svc.ProjectFolderID,
```
Comment (bobheadxi) [inline dev/managedservicesplatform/managedservicesplatform.go:82]: Is this change needed?

File: dev/managedservicesplatform/managedservicesplatform.go [hunk 2]
```diff
@@ -79,12 +79,13 @@ func (r *Renderer) RenderEnvironment(
...
 	})
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to create project stack")
 	}
 	if _, err := cloudrun.NewStack(stacks, cloudrun.Variables{
-		ProjectID:   *projectOutput.Project.Id(),
+		ProjectID:   *projectOutput.Project.ProjectId(),
```
Comment (bobheadxi) [inline dev/managedservicesplatform/managedservicesplatform.go:88]: 🚀

File: dev/managedservicesplatform/managedservicesplatform.go (hunk 1 above)
Comment (unknwon) [reply dev/managedservicesplatform/managedservicesplatform.go:82]: I want to test on a different folder that i don't need to play entitle drama 😂 but i think it's a general useful feature 😉 also the fact that field already exists

**Review by bobheadxi — APPROVED** [review]

**Comment by bobheadxi** [comment]
Depends on fixes in #56608
